source/clear-linux/guides/tooling/mixer.rst

```diff
@@ -339,12 +330,47 @@ set to get a smaller kernel image, which will also be faster to load.
    Look in ~/mixer/update/www/<mix version> to see the update content in your
    workspace.
 
-Example 3: Deploy updates to target
+Example 3: Edit bundle content
+==============================
+
+This example shows how to list the bundles in your mix and edit the content from a bundle. We will remove the `joe` bundle from the `editors` bundle.
+
+#. List the bundles in a mix. To view the includes, add the `--tree` flag.
+
+   .. code-block:: bash
+
+      mixer bundle list
+
+#. Add the `editors` bundle from upstream.
+
+   .. code-block:: bash
+
+      mixer bundle add editors
+
```

Comment:
Add step:
#. Make it a local bundle first before you can modify it.

.. code-block:: bash

   mixer bundle edit editors